articles/systems/02-input-system.md

```diff
@@ -1,23 +1,125 @@
+
+From here you can either remove a controller profile completely, or change which platforms it is active for.
+
+> Be aware, the XRTK will do exactly as it is told in regards to the configuration applied, so if you have two competing controllers active for a specific platform at the same time, you will get multiple controllers in the scene, e.g. OpenVR and Oculus controllers.
+
+## Controller Data Profile configuration
+
+Looking at the configuration for an individual Controller Profile will show you two views:
+
+* Basic (default) - the same simple controller button view as [Controller Actions](#controller-actions) showing only those specific to this provider.
+* Advanced - More detailed views of the configuration behind the controllers.
+
+![](/images/Configuration/InputSystem/ControllerDataProviderProfile.png)
+
+## Controller Profile
+
+When you open a Controller Profile, you will get a nice graphical view of the controller and the inputs available for the device based on its hardware and the SDK of the vendor.  Here you can chose from the list of [Input Actions](#input-actions) you have configured for the project, with each input filtering those Actions by what the Input on the controller can support, for example:
+
+* Buttons can support only `Digitial` actions (boolean), button down and button up. 
+* Triggers support `Single Axis` or `float` data, giving a range, usually between 0 and 1 for how uch the input is pressed.
+* Touch pads and ThumbSticks support `Dual Axis` or `Vector2` data, giving two coordinates in the X and Y planes based on how the input is manipulated
+* We also allow assigning an action to the pose (6dof, position and rotation) of the controller, which relates to the controllers position and rotation in relation to the headset.  An alternate pose is also available for grip, or offset position provided by the vendor for where the user grips the controller.
+
+You may also see a mix of other options where the vendor provides more capabilities, such as Near and Touched events that will show up on those controllers, e.g. Oculus.  Some inputs also have dual options, such as thumbsticks, which can also be pressed, allowing for an alternate action to be assigned.
+
+![](/images/Configuration/InputSystem/ControllerProfile.png)
+
+## Conclusion
+
+There is a lot of configuration available in the Input System, most of which will not need to be touched, but here are some common tasks which can be achieved easily.
+
+* Changing the list of actions specific to your project in the [Input Actions](#input-actions) configuration
+* Altering the Input Action mappings for the registered controllers in [Controller Actions](#controller-actions)
+* Removing controllers that are not needed in [Controller Definitions](#controller-definitions)
 
 ---
 
 ### Related Articles
 
-* [Article 1]()
-* [Article 2]()
-* [Article 3]()
+* [XRTK Platform System](08-platform-framework.md)
```

Comment:
I'd like to see links back to articles that reference this one (like getting started?)

Reply:
Added